src/client/admin.go

```diff
@@ -19,158 +14,3 @@ func (c APIClient) InspectCluster() (*admin.ClusterInfo, error) {
 	}
 	return clusterInfo, nil
 }
-
-// Extract all cluster state, call f with each operation.
```

Comment:
Is the plan to remove Extract/Restore and re-implement it from scratch in 2.0? Do we have a list somewhere of all the things we'll need to get feature parity in 2.0?

Reply:
There is not a plan in place for extract / restore yet. We may end up using the 1.x extract / restore as a reference implementation for what ends up going into 2.0, but for now it is easier to just remove it. We have a TODO list in the 2.0 alpha project, which I am going to add more details to after we get this merged in.